NgsildAgent/util/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+cd /app/util
+
+# Debugging output
+echo "ARG1: $DEVICE_ID"
+echo "ARG2: $GATEWAY_ID"
+echo "ARG3: $KEYCLOAK_URL"
+echo "ARG4: $REALM_ID"
+echo "ARG5: $REALM_USER_PASSWORD"
```

Comment:
Do not output this as default. Make sure it is only printed if DEBUG=true or so.